layouts/css/layout/_sticky-footer.styl

```diff
@@ -9,6 +9,8 @@ body
     flex-direction column
     min-height 100vh
     margin 0
+    overflow-y scroll; /* has to be scroll, not auto */
+    -webkit-overflow-scrolling touch
```

Comment:
Looks like the stylus-auto prefixer plugin was installed?    Is there anything special I need to do to configure it?

> On Dec 9, 2015, at 2:41 PM, Steve Mao <notifications@github.com> wrote:
> 
> In layouts/css/layout/_sticky-footer.styl:
> 
> > @@ -9,6 +9,8 @@ body
> >      flex-direction column
> >      min-height 100vh
> >      margin 0
> > +    overflow-y scroll; /* has to be scroll, not auto */
> > +    -webkit-overflow-scrolling touch
> are we using autoprefixer for this?
> 
> —
> Reply to this email directly or view it on GitHub.
>